### ISSUE TITLE
Add batch norm to default_n_bit_quantize_registry and default_8_bit_quantize_registry

### ISSUE THREAD

**Issue by DerryFitz** [tensorflow/model-optimization#1099]
- TensorFlow version (you are using): 2.13
- 

**Motivation**
There are many models which use batch norm in places where it is not covered by the existing cases allowed in the registry.
Adding batch norm to the registry would allow users used QAT on such models.

At present I am editing both registries by adding the line

_QuantizeInfo(layers.BatchNormalization, ['gamma'], [], True),

which works for my case, but it would be nice to have a more general fix

**Comment by doyeonkim0** [comment]
@Xhark Could you take a look at this issue? Thank you! :)

**Comment by DerryFitz** [comment]
Any update on this?
